default.nix

```diff
@@ -33,6 +34,7 @@ let
         env = pkgs.callPackage ./nix/bolos-env.nix { clangVersion = 7; };
         target = "TARGET_NANOX";
         targetId = "0x33000004";
+        test = false;
```

Comment:
i guess speculos only works with nano s?

Reply:
Yes. I looked at getting it running but there's a fair number of system calls to add.